modules/introduction/partials/new-features-76.adoc

```diff
@@ -122,6 +122,15 @@ You can change the alert period via the new `certExpirationDays` alert limit set
 Couchbase Server sends a second alert when certificates expire.
 See xref:learn:security/certificates.adoc#certificate-expiration[Certificate Expiration] for more information. 
 
+* Couchbase Server now defaults to using the (https://en.wikipedia.org/wiki/Argon2)[Argon2id^] algorithm to hash passwords for new users. 
+This hashing algorithm is more secure than the SHA1 algorithm used to hash passwords in earlier server versions.
+
+* If you upgrade a database to Couchbase Server 7.6 or later, it continues to use the older SHA1 hashing algorithm for existing user passwords.
+You can enable a new setting that has Couchbase Server migrate user passwords from SHA1 to Argon2id when a user authenticates.
+This setting only works if your the entire cluster  is running Couchbase Server version 7.6 or later.
```

Comment:
Edit this sentence to change the placement of "only," fix "if your the cluster" typo, and remove possible extra space after cluster.

This setting works only if the entire cluster is running Couchbase Server version 7.6 or later.